robot.py

```diff
@@ -9,7 +9,8 @@
 
 LEFT_STICK = GenericHID.Hand.kLeft
 RIGHT_STICK = GenericHID.Hand.kRight
-
+LEFT_BUMPER = GenericHID.Hand.kLeft
```

Comment:
Maybe we should change all the `LEFT_BUMPER`, `RIGHT_STICK`, etc. to just `LEFT` and `RIGHT` (or `LEFT_HAND` and `RIGHT_HAND`) since we use them for multiple purposes?

Reply:
Sounds terrific, I'll change that